workspace-manager-clienttests/src/main/java/scripts/uploadscripts/SummariesToBigQuery.java

```diff
@@ -0,0 +1,215 @@
+
+        // insert into testScriptResults
+        tableId = TableId.of(datasetName, testScriptResultsTableName);
+        InsertAllRequest.Builder insertRequestBuilder = InsertAllRequest.newBuilder(tableId);
+        for (int ctr = 0; ctr < renderedTestConfiguration.testScripts.size(); ctr++) {
+            TestScriptSpecification testScriptSpecification =
+                    renderedTestConfiguration.testScripts.get(ctr);
+            TestScriptResult.TestScriptResultSummary testScriptResult =
+                    testRunSummary.testScriptResultSummaries.get(ctr);
+            insertRequestBuilder.addRow(
+                    buildTestScriptResultsRow(testScriptSpecification, testScriptResult));
+        }
+        BigQueryUtils.insertAllIntoBigQuery(bigQueryClient, insertRequestBuilder.build());
+
+        // insert into measurementCollection
+        if (measurementCollectionSummaries == null) {
+            logger.info("No measurement summaries found.");
+        } else {
+            tableId = TableId.of(datasetName, measurementCollectionTableName);
+            insertRequestBuilder = InsertAllRequest.newBuilder(tableId);
+            for (MeasurementCollectionScript.MeasurementResultSummary measurementResult :
+                    measurementCollectionSummaries) {
+                insertRequestBuilder.addRow(buildMeasurementCollectionRow(measurementResult));
+            }
+            BigQueryUtils.insertAllIntoBigQuery(bigQueryClient, insertRequestBuilder.build());
+        }
+    }
+
+    /** Build a single row for each measurement collection. */
+    private Map<String, Object> buildMeasurementCollectionRow(
+            MeasurementCollectionScript.MeasurementResultSummary measurementResult) {
+        Map<String, Object> rowContent = new HashMap<>();
+
+        rowContent.put("testRun_id", testRunSummary.id);
+
+        rowContent.put("description", measurementResult.description);
+
+        rowContent.put("statistics_min", String.valueOf(measurementResult.statistics.min));
+        rowContent.put("statistics_max", String.valueOf(measurementResult.statistics.max));
+        rowContent.put("statistics_mean", String.valueOf(measurementResult.statistics.mean));
+        rowContent.put(
+                "statistics_standardDeviation",
+                String.valueOf(measurementResult.statistics.standardDeviation));
+        rowContent.put("statistics_median", String.valueOf(measurementResult.statistics.median));
+        rowContent.put(
+                "statistics_percentile95", String.valueOf(measurementResult.statistics.percentile95));
+        rowContent.put(
+                "statistics_percentile99", String.valueOf(measurementResult.statistics.percentile99));
+        rowContent.put("statistics_sum", String.valueOf(measurementResult.statistics.sum));
+
+        return rowContent;
+    }
+
+    /** Build a single row for each test script result. */
+    private Map<String, Object> buildTestScriptResultsRow(
```

Comment:
Nit: There's a mixture of snake_case and camelCase naming conventions for the map keys in this function. Example `testRun_id` vs `numberOfUserJourneyThreadsToRun`.
Is that something we can control/change?

Reply:
Looks like there's some consistency with this across functions, but it's not clear to me what the convention is.

Reply:
The snake case refers to the table field names, I think that's the convention familiar to a db developer.
The camel case is common in code like Java.